mantle/platform/api/aws/network.go

```diff
@@ -20,6 +20,8 @@ import (
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/aws/awserr"
 	"github.com/aws/aws-sdk-go/service/ec2"
+
```

Comment:
can this extra line be removed ?

Reply:
It's separated in other files, usually indicating libraries are getting pulled from different locations (i.e. this is a mantle library rather than a vendored one).